api/controller/gateway.go

```diff
@@ -88,6 +89,18 @@ func (g *GatewayStruct) addHTTPRule(w http.ResponseWriter, r *http.Request) {
 			values["private_key"] = []string{"The private_key field is required"}
 		}
 	}
+	if len(req.Rewrites) > 0 {
+		rewrites, err := commonutil.StringToSlice(req.Rewrites)
```

Comment:
Why pass in a string instead of a slice directly

Reply:
What I designed was to store the rewrite data structure directly into the console-db table, so that there is no need to convert to slice for console passed the data to rbd-api.. so what do you think?